qemu/tests/qemu_guest_agent.py

```diff
@@ -2312,7 +2312,7 @@ def _guest_cmd_run(guest_cmd, cmd_args=None, env_qga=None,
     def _action_before_fsfreeze(self, *args):
         session = self._get_session(self.params, None)
         if self.params.get("os_type") == "linux":
-            session.cmd("restorecon -Rv /", timeout=180)
+            session.cmd("restorecon -Rv /mnt", timeout=360)
```

Comment:
Should we don't need to change the timeout here.

Reply:
Updated:

JOB ID     : 6f41b60a7cd9f6f2543ddc6480038f323d158700
JOB LOG    : /root/avocado/job-results/job-2024-02-27T09.02-6f41b60/job.log
Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: PASS